docker/nginx/artemis-server.conf

```diff
@@ -23,6 +23,8 @@ location / {
     fastcgi_send_timeout 900s;
     fastcgi_read_timeout 900s;
     client_max_body_size 128M;
+    # used to advertise the availability of HTTP/3
+    add_header alt-svc 'h3=":443"; ma=2592000,h3-29=":443"; ma=2592000';
```

Comment:
You could consider using `persist=1` here to prevent these entries to be deleted when clearing the network config in the browser.